application/src/test/java/org/thingsboard/server/mqtt/AbstractMqttIntegrationTest.java

```diff
@@ -60,6 +61,30 @@
 
     private static final AtomicInteger atomicInteger = new AtomicInteger(2);
 
+    protected static final String DEVICE_TELEMETRY_PROTO_SCHEMA = "syntax =\"proto3\";\n" +
+            "\n" +
+            "package test;\n" +
+            "        \n" +
+            "message PostTelemetry {\n" +
```

Comment:
Please provide an example that also contains inner object. This will be more useful for the end user (to use as a referene.

Reply:
Same logic should be applied for the "default" input in the UI form.

Reply:
@ashvayka do you mean that I need to add another test case with an inner object, and also add the placeholder to the UI input, or this should be the default value?